app/bundles/CoreBundle/DependencyInjection/MauticCoreExtension.php

```diff
@@ -147,6 +147,10 @@ public function load(array $configs, ContainerBuilder $container)
                                 }
 
                                 $definition->addTag($tag, $tagArguments[$k]);
+
+                                if ('mautic.email_transport' === $tag) {
+                                    $container->setAlias(sprintf('swiftmailer.mailer.transport.%s', $name), $name);
```

Comment:
It can but too easy to forget making one wonder why it's broken. So added this here to prevent requiring devs to set this for custom email transports.